weblate/static/loader-bootstrap.js

```diff
@@ -1380,6 +1380,33 @@ $(function () {
     });
   });
 
+  $("input[name='period']").daterangepicker(
+    {
+      autoApply: true,
+      startDate: moment(),
+      endDate: moment(),
+      alwaysShowCalendars: true,
+      opens: "left",
```

Comment:
Localize custom range label:

```suggestion
      opens: "left",
      locale: {
        customRangeLabel: gettext("Custom range"),
      },
```